examples/Responses/Example02_SimpleResponseStreaming.cs

```diff
@@ -11,6 +11,7 @@ namespace OpenAI.Examples;
 
 public partial class ResponseExamples
 {
+    [Ignore("Test org cannot use streaming")]
```

Comment:
The first thing I did was run every test in main and add Ignore attributes to all failing tests so that I could effectively validate the migration did not affect existing test behavior.